tests/multihost/container-network/data/test.fmf

```diff
@@ -0,0 +1,17 @@
+/server/setup:
+    require: httpd
+    test: |
+        systemctl start httpd
+        echo 'podman rulez' > /var/www/html/index.html
+
+/client:
+    test: |
+        echo "Guest topology stored in '$TMT_TOPOLOGY_BASH':"
+        cat "$TMT_TOPOLOGY_BASH"
+        . "$TMT_TOPOLOGY_BASH"
+
+        for count in {1..33}; do
```

Comment:
Probably not needed. This repeated fetch and random sleep are interesting for output of two parallel clients. Let's speed up the test a bit.